### PR TITLE
Data mocking using proxy

### LINKED ISSUE #58: Better data mocking
The integration tests use `cy.intercept()` to mock back-end data, but because of SSR it's not possible to intercept many of the API calls being made (from the back-end). This means that to avoid 404's the initial requests made by `cy.visit()` must be for a resource that can be accessed via the live (or dev) API, which changes over time, making the tests brittle.

Consider running a small API proxy for integration testing, and substituting it by re-configuring the devserver under test. See #11 for API externalization.

### PR DESCRIPTION
**Pull request by richardolsson**
This PR introduces data mocking using an API proxy sitting between the app and the Zetkin API.

The proxy is an instance of our new little open-source project [moxy – the mocking proxy](https://github.com/zetkin/moxy). 

Here's how it works:

1. The app when started is configured to use moxy instead of the Zetkin API
2. Moxy is configured to forward requests by default to the Zetkin API
3. The test setup (running in cypress) configures mocks through moxy's REST API
4. The test runs and "drives" the app
5. The app makes requests to the API (i.e. moxy) as usual
6. Any mocked requests return the mock data, all other requests are forwarded to the Zetkin API
7. The test asserts data is rendered to DOM correctly

Tests will soon be able to verify that requests were made through moxy's REST API (not currently implemented) using it's upcoming _spies_ feature.

```
  [ NEXT.js server ]---------------[ moxy ]-------------[ Zetkin API ]
         |                             |
         |                             |
[ React browser app ]             set up mocks &
         |                       verify requests
     drive app                         |
         |                             |
  [ Cypress tests ]--------------------'
```

Resolves #58

### CODE REVIEW

File: cypress/integration/org_events_page.spec.ts [hunk 1]
```diff
@@ -1,22 +1,51 @@
...
+                .findByText('More info')
+                .click();
+            cy.url().should('match', new RegExp(`/o/1/events/${json.data[0].id}$`));
+        });
     });
 
-    xit('contains a placeholder if there are no events', () => {
-        // TODO: Figure out why this fails on GitHub
+    it('contains a placeholder if there are no events', () => {
+        cy.request('put', 'http://localhost:8001/v1/orgs/1/campaigns/1/actions/_mocks/get', {
```
Comment (niklasva82) [inline cypress/integration/org_events_page.spec.ts:41]: This test doesn't actually try to find any placeholder?

Comment (richardolsson) [reply cypress/integration/org_events_page.spec.ts:41]: Yes, I believe it does, on line 52.

Although I found another issue revisiting the code now, which is that the old intercept is still left behind and should be removed.

Comment (niklasva82) [reply cypress/integration/org_events_page.spec.ts:41]: ah ok! unfmailiar with cypress and must have missed it. Looks good!

**Review by niklasva82 — APPROVED** [review]

**Comment by richardolsson** [comment]
Approved on Slack. Merging!

**Pull request merged**